src/python/platforms/linux/lkl/kernel_utils.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Linux Kernel Library kernel utils functions."""
+import os
+
+from base import utils
+from platforms.android import symbols_downloader
+from system import environment
+
+from . import constants
+
+
+def _should_download_symbols():
+  """Return Ture if we should continue to download symbols."""
+  # For local testing, we do not have access to the cloud storage bucket with
+  # the symbols. In this case, just bail out.
+  if environment.get_value('LOCAL_DEVELOPMENT'):
+    return False
+
+  # When running reproduce tool locally, we do not have access to the cloud
+  # storage bucket with the symbols. In this case, just bail out.
+  if environment.get_value('REPRODUCE_TOOL'):
+    return False
+
+  return True
```

Comment:
nit: this can be all simplified as:

```python
return (not environment.get_value('LOCAL_DEVELOPMENT') and 
        not environment.get_value('REPRODUCE_TOOL'))
```